### PR TITLE
ci: fix coverage

### PR DESCRIPTION
**Pull request by lizan**
Signed-off-by: Lizan Zhou <lizan@tetrate.io>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -47,6 +47,7 @@ jobs:
 
    coverage:
      executor: ubuntu-build
+     resource_class: 2xlarge
```
Comment (ggreenway) [inline .circleci/config.yml:50]: Should this be in a separate PR from the build fix?

Comment (lizan) [reply .circleci/config.yml]: Sigh, seems we don't have 2xlarge available in our circle account. But I think we can do this in same PR.

**Comment by ggreenway** [comment]
@lizan coverage failed in `SniDynamicProxyFilterTest.LoadDnsCache`
